meorg_client/client.py

```diff
@@ -442,6 +442,31 @@ def start_analysis(self, id: str) -> Union[dict, requests.Response]:
             url_params=dict(id=id),
         )
 
+    def post_output_entity(
```

Comment:
The name `post_output_entity` doesn't follow the established naming convention.

Suggest something like:

```suggestion
    def model_output_create(
```

Reply:
The general idea (although I've failed to follow this myself on occasion) is:

`{THING}_{VERB}`, where VERB is anything other than a standard getting of the details.

So a model_output will eventually have:
- `model_output` - For getting details about a model output and usually operating over http `GET`
- `model_output_create` - For creating one and usually operating over http `POST`
- `model_output_delete` - For deleting one and usually operating over http `DELETE`.

There are the odd use-cases for http `PATCH` in the case of an update, but I don't think that will impact this part of the code.